apps/api/src/domain/models/book/entities/book.entity.ts

```diff
@@ -1,11 +1,17 @@
 import { PartialDTO } from 'apps/api/src/types/partial-dto';
 import { resourceTypes } from '../../../types/resourceTypes';
+import { EdgeConnectionContextType } from '../../context/types/EdgeConnectionContextType';
 import { Resource } from '../../resource.entity';
 import BookPage from './BookPage';
 
 export class Book extends Resource {
     readonly type = resourceTypes.book;
 
+    readonly allowedContextTypes = [
```

Comment:
We might not want this here. How could we read this statically if we put it here?